node/.prettierignore

```diff
@@ -1,6 +1,5 @@
 # ignore that dir, because there are a lot of files which we don't manage, e.g. json files in cargo crates
 rust-client/*
-*.md
```

Comment:
Please revert

Reply:
@adarovadya can you explain why this was added to the PR?  I see it was you that added the ignore originally.